src/refresh.ts

```diff
@@ -3,32 +3,32 @@ import * as util from './utilities';
 import * as state from './state';
 import { NReplSession } from './nrepl';
 import * as replSession from './nrepl/repl-session';
+import { appendLineEvalOut } from './results-output/output';
```

Comment:
I know it is not consistent across the code base (and not even in this file), but I do prefer aliasing the imports. I think that what is consistent is that we alias the output module to `output` everywhere.

Reply:
So, since I said I will merge in a comment above, let me clarify 😄  I'll merge once this is addressed.